### DISCUSSION TITLE
Using External Drive to Store Videos to be Annotated

### DISCUSSION THREAD

**Discussion by hfreeman02** [biigle/core#843] [Q&A] [answered]
Hello,

I am currently loading videos in Biigle using the Storage Request option. I know that I will max out the 100GB storage shortly so I am looking for alternatives. Is it possible to link Biigle to an external hard drive that the files are stored on so that I can avoid doing Storage Requests and have more storage space available?

Thanks!

**Comment by mzur** [comment, marked as answer]
You have two options:

1. On biigle.de the files must be online somewhere. A local hard drive won't work. But you can write us at info@biigle.de, briefly describing your research project and the amount of storage you need, and we will most likely extend your storage quota.

2. You self-host BIIGLE on your own machine (see [here](https://github.com/biigle/biigle) for the installation instructions). Self-hosting is not easy but with BIIGLE running on your machine, you can also load locally stored files.

**Reply by draker42** [reply]
Hi Martin, 
I am the IT support here, and I have successfully completed the installation of BIIGLE on a local debian server, following the instructions. I hope this will solve all the large data issues our team is encountering. I may ask some new questions...
Thanks, 
-D.
